wireless/bluetooth/bt_atomic.c

```diff
@@ -38,10 +44,10 @@ bt_atomic_t bt_atomic_incr(FAR bt_atomic_t *ptr)
   irqstate_t flags;
   bt_atomic_t value;
 
-  flags = spin_lock_irqsave(NULL);
+  flags = spin_lock_irqsave_wo_note(&g_bt_atomic_lock);
```

Comment:
why not replace to atomic_* api directly

Reply:
done